test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.json

```
@@ -0,0 +1,392 @@
+{
+  "contracts" : 
+  {
+    "test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.sol:DeprecatedVar" : 
```

```suggestion
    "deprecated_untyped_variable_declaration.sol:DeprecatedVar" : 
```
Here also should be changed.

---

@s0b0lev thanks for review and feedback. I've applied some changes due to raised remarks. Please, reconsider.